components/feature/addons/src/main/java/mozilla/components/feature/addons/ui/AddonsManagerAdapter.kt

```diff
@@ -167,6 +166,7 @@ class AddonsManagerAdapter(
         holder: UnsupportedSectionViewHolder,
         section: NotYetSupportedSection
     ) {
+        val unsupportedAddons = addonsMap.values.filter { it.inUnsupportedSection() }
```

Comment:
nit: maybe add test for when we are creating sections, we have the correct unsupportedAddons list?

Reply:
We have already have one :)
https://github.com/mozilla-mobile/android-components/blob/01cfc5e581678fe6ed9700c5872711cd2967eeda/components/feature/addons/src/test/java/mozilla/components/feature/addons/ui/AddonsManagerAdapterTest.kt#L58